morepath/app.py

```diff
@@ -17,16 +17,53 @@
 
 import dectate
 
+from webob.exc import HTTPNotFound
 from .request import Request
 from . import compat
-from .implicit import set_implicit
 from .reify import reify
-from . import generic
 from .path import PathInfo
 from .error import LinkError
+from .dispatch import delegate, reg
+from .compat import with_metaclass
 
 
-class App(dectate.App):
+class Config(object):
```

Comment:
Indeed. I migrated to the Dectate library from repository and removed the Metaclass.